lib/manifestTools/assets/web-manifest-extended.json

```diff
@@ -11,7 +11,43 @@
       "items": {
         "$ref": "#/definitions/mjs_access_whitelist_rule"
       }
-    }
+    },
+    "mjs_extended_scope": {
+      "description": "The mjs_extended_scope member is an array of rules that define the navigation scope of this web application's application context.",
+      "type": "array",
+      "items": {
+        "type": "string", "format": "uri" 
+      }
+    },
+    "mjs_api_access": {
+      "description": "The mjs_api_whitelist member is an array of rules that indicates the type of access to the runtime API the URLs matching the rule should have.",
+      "type": "array",
+      "items": {
+        "$ref": "#/definitions/mjs_api_access_rule"
+      }
+    },
+    "mjs_import_scripts": {
+      "description": "The mjs_import_scripts member is an array of script files that will be loaded into the application.",
+      "type": "array",
+      "items": {
+        "$ref": "#/definitions/mjs_import_scripts_rule"
+      }
+    },
+    "mjs_cordova": {
+      "description": "Specify how to load the Cordova and plugin JavaScript files.",

```

Comment:
Specify how to load the Cordova plugin and its associate resources
Specify how to load the Cordova plugin and its associate JavaScript sources